src/main/java/com/flipkart/foxtrot/client/senders/HttpSyncEventSender.java

```diff
@@ -88,17 +88,10 @@ public void send(final String table, byte[] payload) {
             }
         } catch (FeignException e) {
             logger.error("table={} msg=event_publish_failed", new Object[]{table}, e);
```

Comment:
When we suspended foxtrot yesterday, we were just logging this exception and fabirc topology continued